### PR TITLE
build: add build:qe script for pr_check

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

**Comment by ezr-ondrej** [comment]
Still failing :(

**Comment by amirfefer** [comment]
as expected @ezr-ondrej, the routes also need to change in qe tests

**Comment by ezr-ondrej** [comment]
Could you add in comment what the new routes are? I guess we can merge this, so QE can try this out :)

File: fec.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const { resolve } = require('path');
 
 module.exports = {
-  appUrl: '/insights/provisioning',
+  appUrl: '/staging/starter',
```
Comment (amirfefer) [inline fec.config.js:5]: DO NOT merge this! just a pipeline test

**Comment by amirfefer** [comment]
this workaround failed to work, closing

**Pull request closed without merging**